devops.rst

```diff
@@ -14,6 +14,7 @@ If you plan to use Conan in production in your project, team, or organization, t
 
    devops/using_conancenter
    devops/devops_local_recipes_index
+   devops/continuous_integration/tutorial
```

Comment:
Maybe we can use the `guide` instead `tutorial` to avoid confusion with the `tutorial` section?

Reply:
or moving it to the tutorial would be another option?

Reply:
I agree that guide might be a tad less confusing. Also, I'd set this as the first item in the devops tab